### PR TITLE
DDPB-3919 update dependencies and fix activation issues

### PR DESCRIPTION
**Pull request by jamesrwarren**
## Purpose
Whilst looking into the activation issues, a few things came up. This addresses some things:

- Issues caused by double clicking it seems (probably better way of stopping this but added some tags around the submits to stop it)
- there was a missing template
- there was a template issue where we're calling characters letters
- Also chucking in an upgrade to dependency that dependabot mentioned

Fixes DDPB-3919

## Approach
NA

## Learning
The activation issue was very tricky to work out but obvious in hindsight (shout out to Charles for his help). 
The webpack upgrade was a bit picky. We can't upgrade node any further with the versions of packages we have. There's also a fun error based on the docker version being used by circle for remote_docker images that means this is the package selection chosen...

## Checklist
- [ ] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [ ] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [ ] Translations are used and the profiler doesn't identify any missing
- [ ] Any links or buttons added are screen reader friendly and contextually complete
- [ ] If adding GA events, I have updated or [checked](docs/runbooks/GOOGLE-ANALYTICS.md) the existing category or label values

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "standard": "^14.3.4",
     "stickyfilljs": "^2.1.0",
     "url-loader": "^3.0.0",
-    "webpack": "^4.44.2",
+    "webpack": "^5.24.4",
```
Comment (acsauk) [inline client/package.json:32]: I'm a little bit wary of jumping to a new major version without doing some checks on if any JS features break. Could we go to 4.46 instead? That should update the elliptical dependabot warning.

File: client/src/Controller/AbstractController.php [hunk 2]
```diff
@@ -14,7 +14,7 @@ abstract class AbstractController extends Controller
      */
     protected function renderError(string $description, $statusCode = 500)
     {
-        $text = $this->renderView('TwigBundle/Exception/template.html.twig', [
+        $text = $this->renderView('bundles/TwigBundle/Exception/template.html.twig', [
```
Comment (acsauk) [inline client/src/Controller/AbstractController.php:17]: Oh snap was this broken the whole time? Sure I had tested this locally when I made the change 😭

File: client/package.json (hunk 1 above)
Comment (jamesrwarren) [reply client/package.json]: I did do some checks and it all looked fine but now you've said that I don't want egg on my face when something breaks so putting it to that suggested version ;)

**Review by acsauk — APPROVED** [review]
Good detective work @jamesrwarren and @charlesmmarshall - I'm thinking we should find a way to update all of our buttons to have this on by default as when is it ever a good thing to make two form submission? Will add a ticket to the backlog :shipit:

**Pull request merged**